packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -43,6 +43,11 @@ class UpgradeCommand extends FlutterCommand {
     if (code != 0)
       return code;
 
+    // Causes us to update our locally cached packages.
+    code = await runCommandAndStreamOutput(<String>[
+      'bin/flutter', '--version'
```

Comment:
Yes, although in this case printing out the version actually makes sense because its showing you which version of flutter you've upgraded to.